subiquitycore/core.py

```diff
@@ -130,12 +130,18 @@ def make_palette(colors, styles):
 
     urwid_palette = []
     for name, fg, bg in styles:
-        urwid_palette.append((name, urwid_name[fg], urwid_name[bg]))
+        urwid_fg, urwid_bg =  urwid_name[fg], urwid_name[bg]
+        if ascii:
```

Comment:
Could you add some comments here? I guess what's going on is that there are some palette choices that work ok in 24 bit mode but look crap with the traditional colors?